examples/tutorials/nb_python/Habitat2_Quickstart.py

```diff
@@ -435,18 +436,19 @@ class NavPickSuccessMeasurementConfig(MeasurementConfig):
     action_space_config: v0
     concur_render: False
     auto_sleep: False
-    agent_0:
-      height: 1.5
-      is_set_start_state: False
-      radius: 0.1
-      sim_sensors:
-        head_rgb_sensor:
-          height: 128
-          width: 128
-      start_position: [0, 0, 0]
-      start_rotation: [0, 0, 0, 1]
-      robot_urdf: ./data/robots/hab_fetch/robots/hab_fetch.urdf
-      robot_type: "FetchRobot"
+    agents:
+      main_agent:
```

Comment:
Okay, so why are we naming the agent with dictionary keys instead of just introducing a harmless name object to the AgentConfig? Seems like we are just complicating multiagent support. Is there a good reason to do this? Is it due to limitations in YAML? That dictionaries are easier to merge together? Can we preserve the ordering of the dictionary somehow to specify agent order?

Reply:
Better yet can we copy the way defaults is defined at the top fo the yaml
```suggestion
    agents:
        - main_agent: ...
        - agent_1: ...
```